src/core/adapters/notion.ts

```diff
@@ -41,7 +41,7 @@ function uuid(slugId: string) {
 }
 
 function getPageFromPath(path: string) {
-  const { slug } = match("/:organization/:slug", path);
+  const slug = path.split("/").slice(-1)[0];
```

Comment:
Let's keep using the `micro-match` package here:

```js
const { slug } = match(":organization?/:slug", path);
```

It can do optional parameters.